SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -98,13 +98,6 @@ bool SpatialVirtualWorkerTranslator::IsValidMapping(Schema_Object* Object) const
 // a worker first becomes authoritative for the mapping.
 void SpatialVirtualWorkerTranslator::ApplyMappingFromSchema(Schema_Object* Object)
 {
-	if (!IsValidMapping(Object))
```

Comment:
I don't believe this check is necessary anymore - was originally added https://github.com/spatialos/UnrealGDK/commit/f07850cd89746a4cc04063e138894ac0804d10b1

Reply:
I think it would still be worth validating that the mapping matches the number of workers expected for the deployment, although I see that you've got that covered elsewhere too.

Reply:
Yep, additionally the ValidMapping was only returning true if a of the physical worker names in the data matches that of the local worker, which isn't valid for snapshot reloading.

Reply:
IsValidMapping can be removed now, right?